### ISSUE TITLE
No shutdown on newest version of Fenen os

### ISSUE THREAD

**Issue by arkjohn5727** [feren-OS/Bug-Reporting-Center#231]
**Package Name or Section of Feren OS where this bug resides**
Where does this bug happen? (e.g.: feren-themer-cinnamon) ___

**Describe the bug**

**Steps to Reproduce**
1. 

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by arkjohn5727** [comment]
I just installed Feren today and if went flawlessly. HOWEVER.. I canot get the system to either reboot or shurdown. The only remedy i have is to use the terminal or hold down the powed buttonon my laptop to force a shutdown. I did  reboot then went into recovery nmode and did the fist broken packages and then the repair grup boot but nothing has worked so far.
Any suggestion  would be greatly apreciated

FYI i have Feren installed on a THinkpad T430   and onto a msata hdd

Thanks again  to any all that may reply to this
